packages/timeline-state-resolver/src/conductor.ts

```diff
@@ -896,26 +909,21 @@ export class Conductor extends EventEmitter<ConductorEvents> {
 
 			// Push state to the right device:
 			await this._mapAllDevices(false, async (device: DeviceContainer<DeviceOptionsBase<any>>): Promise<void> => {
+				if (this._options.optimizeForProduction) {
```

Comment:
Wouldn't it be better to use the conventional `NODE_ENV` environment variable for this?

Reply:
Ideally, yes it would. I intended this PR to be a minimal change, so consider this being a feature flag (because it botches the unit tests that are testing the abstract device).

If this turns out to be a good solution I believe that we should eventually remove the ABSTRACT devices altogether (ie keep the timeline-objects, but never spawn devices).

Reply:
Feel free to merge then.